src/otg/method/clump.py

```diff
@@ -32,17 +32,17 @@ def _is_lead_linked(
             variant_id (Column): Lead variant id
             p_value_exponent (Column): p-value exponent
             p_value_mantissa (Column): p-value mantissa
-            credible_set (Column): Credible set <array of structs>
+            locus (Column): Credible set <array of structs>
 
         Returns:
             Column: Boolean in which True indicates that the lead is linked to another tag in the same dataset.
         """
         leads_in_study = f.collect_set(variant_id).over(Window.partitionBy(study_id))
         tags_in_studylocus = f.array_union(
             # Get all tag variants from the credible set per studyLocusId
-            f.transform(credible_set, lambda x: x.tagVariantId),
+            f.transform(ld_set, lambda x: x.tagVariantId),
             # And append the lead variant so that the intersection is the same for all studyLocusIds in a study
-            f.array(f.col("variantId")),
+            f.array(variant_id),
```

Comment:
How did it work in the original implementation when it was:
```python
f.array(f.col("variantId")),
```
however `variant_id` was a column object already?

Reply:
It was working, but the column name was hardcoded. The function parameter was ignored. @ireneisdoomed spotted this one